plugins/vpp/l2plugin/vppcalls/arp_term_vppcalls.go

```diff
@@ -33,7 +33,11 @@ func (h *BridgeDomainVppHandler) callBdIPMacAddDel(isAdd bool, bdID uint32, mac
 	if err != nil {
 		return err
 	}
-	req.Mac = l2ba.MacAddress{Bytes: macAddr}
+	var macInt [6]uint8
```

Comment:
This does not have to be so complex. You can do the same using single call `copy(req.Mac[:], macAddr)`.